### ISSUE TITLE
[BUG] UnusedCustomPropertiesRule false positive on whole-object param/custom forwarding

### ISSUE THREAD

**Issue by alex-lopez-bwdg** [bw-design-group/ignition-lint#107]
**Describe the bug**

`UnusedCustomPropertiesRule` false positives on view params and view-level custom properties that are forwarded as a whole object rather than accessed by member name.

When a script passes the whole params object to a consumer:

```python
system.perspective.sendMessage('onMoveTagUpClicked', scope='page', payload=self.view.params)
```

...and a receiving view reads members off that payload (`payload.get('sectionUuid')`), none of the params get credited as used. The detection regex requires a trailing `.member`:

```python
nested = r'([a-zA-Z_][a-zA-Z0-9_]*(?:\.[a-zA-Z_][a-zA-Z0-9_]*)*)'
patterns = [
    rf'self\.view\.params\.{nested}',   # requires self.view.params.SOMETHING
    ...
]
```

A bare `self.view.params` produces zero matches, so every param forwarded this way gets flagged unused.

**To Reproduce**

1. Define a view with an input param, e.g. `sectionUuid`.
2. In a button's `onActionPerformed` script, forward the whole params object:

```python
system.perspective.sendMessage('onRemoveTagClicked', scope='page', payload=self.view.params)
```

3. In the receiving view's message handler, read a member: `payload.get('sectionUuid')`.
4. Run ign-lint on the first view with `UnusedCustomPropertiesRule` enabled.
5. See: `params.sectionUuid: view parameter 'sectionUuid' is defined but never referenced`

**Expected behavior**

A bare `self.view.params` or `self.view.custom` reference (no trailing `.member`) should credit all params/custom props at that scope as used. The linter has no way to know which members the consumer will pull off the forwarded object.

**Actual output**

```
• params.sectionUuid: view parameter 'sectionUuid' is defined but never referenced
```

**Additional context**

The pattern occurs when a view forwards its whole params object via `sendMessage` and a receiving view's message handlers read individual members off the payload. The linter sees no `.member` access in the sending view and flags every param as unused, even though they are actively consumed.

Same class of false positive as #97 (parent property not credited when only children are referenced) and #48 (output params consumed cross-view).
